.github/ISSUE_TEMPLATE/3-doc.yml

```diff
@@ -0,0 +1,106 @@
+name: 📄 Documentation Issue
+description: Report an issue with the documentation
+title: "[Doc]: <Write a concise title here>"
+body:
+    - type: checkboxes
+      id: no-duplicate-issues
+      attributes:
+          label: Is there an existing issue for this?
+          description:
+              Please search to see whether the issue you are encountering has already been reported.
+          options:
+              - label:
+                    I have searched the existing
+                    [issues](https://github.com/trvswgnr/sloth-pipe/issues)
+                required: true
+    - type: markdown
+      attributes:
+          value: |
+              Thanks for taking the time to report this issue!
+    - type: input
+      id: contact
+      attributes:
+          label: Contact Details
+          description: How can we get in touch with you if we need more info?
+          placeholder: ex. email@example.com
+      validations:
+          required: false
```

Comment:
This part as well should be removed, as previously mentioned in the past 2 templates.